gps/app/locales/ml.json

```diff
@@ -19,15 +24,21 @@
   },
   "import": {
     "button_text": "പഴയ ലൊക്കേഷനുകൾ ഇംപോർട്ട് ചെയ്യുക",
+    "error": "നിങ്ങളുടെ ഡാറ്റ ഇമ്പോർട്ടുചെയ്യുമ്പോൾ എന്തോ തെറ്റായി സംഭവിച്ചു.",
     "google": {
+      "already_imported": "നൽകിക്കഴിഞ്ഞ് ടേക്ക് ഔട്ട് ഫയൽ ഇതിനകം തന്നെ ഇറക്കുമതി ചെയ്തിട്ടുണ്ട്.",
       "disclaimer": "COVID സെയ്ഫ് പാത്ത്-ന് Google മായി ഒരു ബന്ധവുമില്ല കൂടാതെ നിങ്ങളുടെ ഡാറ്റ ഒരിക്കലും പങ്കിടില്ല.",
-      "instructions_detailed": "ഗൂഗിൾ ടേക്ഔട്ട് സന്ദർശിക്കുക, ഇനിപ്പറയുന്ന ക്രമീകരണങ്ങൾ ഉപയോഗിച്ചുകൊണ്ട് നിങ്ങളുടെ ലൊക്കേഷൻ ചരിത്രം എക്സ്പോർട്ട് ചെയ്യുക: \n1. ഡെലിവറി രീതി: \"ഡ്രൈവ് ചെയ്യാൻ ചേർക്കുക\" \n2. ആവൃത്തി: \"ഒരിക്കൽ എക്സ്പോർട്ട് ചെയ്യുക\" \n3. ഫയൽ തരം & വലിപ്പം: \". തപാൽ\", \"1GB\"\n4. കയറ്റുമതി തയ്യാറാകുമ്പോൾ ഗൂഗിൾ ഒരു ഇമെയിൽ അയയ്ക്കുന്നു \n5. ലൊക്കേഷനുകള് ഇറക്കുമതി ചെയ്യാന് ഇവിടെ തിരിച്ചെത്തും. ഇംപോര് ട്ട് ഉപാധികള്:\n-ഗൂഗിൾ ഡ്രൈവിൽ നിന്ന് ഇംപോർട്ട് ചെയ്യുക\n-ബ്രൗസറിൽ നിന്ന് ഡൗൺലോഡ് ചെയ്യുക, തുടർന്ന് പ്രാദേശിക ഫോൺ ഫയലുകളിൽ നിന്ന് ഇറക്കുമതി ചെയ്യുക. ഫയലുകൾ വലുതാക്കാം എന്നതിനാൽ വൈഫൈ നെറ്റ് വർക്കിലേക്ക് ഉറപ്പാക്കുക.",
+      "file_open_error": "ഫയൽ തുറക്കാൻ കഴിഞ്ഞില്ല. \n ദയവായി, Google ഡ്രൈവിൽ നിന്ന് ഫയൽ തുറന്നിട്ടുണ്ടെന്ന് ഉറപ്പാക്കുക",
```

Comment:
I think there's just one space before the line break in the english file and no space after it. Don't know if that affects how it looks in the app.
കഴിഞ്ഞില്ല. \n ദയവായി,

Reply:
Fix these in lokalise. If we need very specific formatting, add it to the comment in lokalise so that editors maintain specific rules.